content/en/monitors/service_level_objectives/_index.md

```diff
@@ -77,6 +76,34 @@ Setting a 100% target means having an error budget of 0% since error budget is e
 
 To edit an SLO, hover over the SLO's row in the list view and click the edit pencil icon that appears at the right of the row, or click on the row to open the details side panel and select the edit button from the cog icon in the top right of the panel.
 
+## Permissions
+
+#### Role based access
```

Comment:
Capability based access controls

Reply:
@kaixintai I don't see this used in any other product for Permissions features, let me know if I missed this somewhere. Is this specific to SLOs?

Reply:
It was the terminology I was advised to use by @sidph. Sid, is this ok or is there something else you'd like to call resource RBAC?

Reply:
Also Esther, we're blocked by the production freeze, so we haven't enabled granular RBAC for customers yet. Will plan to do it next week and ping you when it's on.